addons/portal_rating/static/src/scss/portal_rating.scss

```diff
@@ -39,12 +36,17 @@ $o-w-rating-star-color: #FACC2E;
             font-size: $font-size-sm;
         }
         .o_website_rating_table_reset {
-            .o_website_rating_select_text {
-                visibility: hidden;
+            .o_website_rating_selection_reset {
+                color: $red;
             }
         }
+        .o_darken_5 {
+            opacity: 0.5;
+        }
+        .o_pointer {
```

Comment:
For pointer, a 
```css
.o_website_rating_table_row:not(o_website_rating_table_row_selected)
``` 
as you did before could still be used, since you do need CSS.

Reply:
Yes it is better, I have changed